### PR TITLE
Sapulpa 2024-01-29

### PR DESCRIPTION
**Pull request by yakra**
https://forum.travelmapping.net/index.php?topic=5496
**OK33:** East end truncated from US 75 Alt and OK 97 to OK 117.
**OK66:** Removed from Dewey Ave and relocated onto OK 117 & OK 117A (Mission St) between waypoints DewAve_W & DewAve_E.

Ping @dietrichmd @bejacob _@mapcat @si404_ @bhoward @ozarkman417 _@theradison @scott5114_ @ua747sp
...and a few others. Something's changed at GitHub and now my script won't pull usernames 100% of the time. :(

### CODE REVIEW

**Comment by dietrichmd** [comment]
so explain that to me like i'm 5...does this mean I need to change my submission method?

**Comment by rickmastfan67** [comment]
> so explain that to me like i'm 5...does this mean I need to change my submission method?

It means labels changed that affect your list file for those routes mentioned above.  You should double check to make sure your mapping is still accurate.

**Comment by ua747sp** [comment]
old routing was in red, so i'm in the process of removing my travels
between the two points circled in blue. based on the forum, looks like
there might be a historic US66 route drawn between those two points along
the red, so will check back periodically until that's added.

[image: image.png]

**Comment by rickmastfan67** [comment]
@ua747sp FYI, images don't work as attachments if you send via email.  You need to post them via Github itself.

**Comment by ua747sp** [comment]
Thanks. Let me try again here.

image
